package.json

```diff
@@ -30,16 +30,10 @@
     "@total-typescript/tsconfig": "^1.0.4",
     "@types/js-search": "^1.4.4",
     "@types/node": "^20",
-    "@types/react": "npm:types-react@19.0.0-rc.1",
-    "@types/react-dom": "npm:types-react-dom@19.0.0-rc.1",
+    "@types/react": "~19.1.2",
+    "@types/react-dom": "~19.1.2",
     "eslint": "^8",
     "eslint-config-next": "15.1.9",
     "typescript": "^5"
-  },
-  "pnpm": {
-    "overrides": {
-      "@types/react": "npm:types-react@19.0.0-rc.1",
-      "@types/react-dom": "npm:types-react-dom@19.0.0-rc.1"
-    }
```

Comment:
this was a workaround from when react 19 was in RC - this is no longer needed now that react 19 is stable